### PR TITLE
Implementing of FAT-7006

### PR DESCRIPTION
**Pull request by IhorBohdan**
***
[FAT-7006](https://issues.folio.org/browse/FAT-7006)
***
[С380708](https://foliotest.testrail.io/index.php?/cases/view/380708)
***
[Allure](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/3722/allure/)
***

### CODE REVIEW

**Review by Sherzod-Kenjaev — APPROVED** [review]

File: cypress/support/fragments/finance/funds/funds.js [hunk 1]
```diff
@@ -102,6 +102,29 @@ export default {
...
+    cy.get('[data-test-col-transfer-from="true"] ul[role="listbox"]')
+      .contains(firstFund.name)
+      .click();
+    cy.get('[data-test-col-transfer-to="true"]').click();
+    cy.get('[data-test-col-transfer-to="true"] ul[role="listbox"]')
+      .contains(secondFund.name)
+      .click();
+    cy.do([
+      cancelButton.click(),
+      Button('Close without saving').click()
```
Comment (nayimovag) [inline cypress/support/fragments/finance/funds/funds.js:123]: add to const

File: cypress/support/fragments/finance/funds/funds.js [hunk 2]
```diff
@@ -102,6 +102,29 @@ export default {
...
+      codeField.fillIn(defaultFund.code),
+      ledgerSelection.open(),
+      SelectionList().select(defaultLedger),
+    ]);
+    cy.get('[data-test-col-transfer-from="true"]').click();
+    cy.get('[data-test-col-transfer-from="true"] ul[role="listbox"]')
+      .contains(firstFund.name)
+      .click();
+    cy.get('[data-test-col-transfer-to="true"]').click();
+    cy.get('[data-test-col-transfer-to="true"] ul[role="listbox"]')
```
Comment (nayimovag) [inline cypress/support/fragments/finance/funds/funds.js:118]: add TODO to refactor it with interactors later

**Review by nayimovag — APPROVED** [review]

File: cypress/support/fragments/finance/funds/funds.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/support/fragments/finance/funds/funds.js]: done

File: cypress/support/fragments/finance/funds/funds.js (hunk 2 above)
Comment (IhorBohdan) [reply cypress/support/fragments/finance/funds/funds.js:120]: Done

**Pull request merged**